src/Microsoft.Diagnostics.Monitoring.RestServer/ProcessFilterOptions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.UnitTests.Options
+#else
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+#endif
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum ProcessFilterKey
+    {
+        Pid,
+        RuntimeId,
+        Name,
+        CommandLine,
+    }
+
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum ProcessFilterType
+    {
+        [Display(Description = "Performs a case-insensitive string comparison.")]
+        Exact,
+
+        [Display(Description = "Performs a case-insensitive substring search.")]
+        Loose,
```

Comment:
Subset? Contains?

Reply:
I initially intended for these to be vague in case we have some other criteria, but I think that's unlikely anyway.